### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by SebastianOpriel**

### CODE REVIEW

File: connector/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 
 ARG BUILD_ARGS
 
 COPY --chown=gradle:gradle . /home/gradle/project/
 WORKDIR /home/gradle/project/
 RUN --mount=type=cache,target=/home/gradle/.gradle/caches gradle build --no-daemon $BUILD_ARGS
 
-# -buster is required to have apt available
-FROM openjdk:17-slim-buster
+FROM eclipse-temurin:17-alpine
```
Comment (sybereal) [inline connector/Dockerfile:9]:
```suggestion
FROM eclipse-temurin:17-jre-alpine
```

The runtime layer shouldn't need a full JDK. Reduces base image size from ~200 MB to ~50 MB.

File: connector/Dockerfile [hunk 2]
```diff
@@ -19,8 +18,8 @@ ENV EDC_BUILD_DATE=$EDC_BUILD_DATE_ARG
 ARG JVM_ARGS=""
 
 # Install curl, then delete apt indexes to save image space
-RUN apt update \
-    && apt install -y curl \
+RUN apk update \
+    && apk add curl \
     && rm -rf /var/cache/apt/archives /var/lib/apt/lists \
```
Comment (sybereal) [inline connector/Dockerfile:23]:
```suggestion
RUN apk add --no-cache curl \
```

File: connector/Dockerfile [hunk 3]
```diff
@@ -19,8 +18,8 @@ ENV EDC_BUILD_DATE=$EDC_BUILD_DATE_ARG
 ARG JVM_ARGS=""
 
 # Install curl, then delete apt indexes to save image space
```
Comment (sybereal) [inline connector/Dockerfile:20]:
```suggestion
```

**Review by sybereal — CHANGES_REQUESTED** [review]

**Comment by tmberthold** [comment]
Closed in favor of https://github.com/sovity/edc-extensions/pull/450

**Pull request closed without merging**